src/mem/port.hh

```diff
@@ -287,15 +323,20 @@ class SlavePort : public Port, public AtomicResponseProtocol,
      * and return a populated list with at least one item.
      *
      * @return a list of ranges responded to
+     *
+     * @ingroup api_port
      */
     virtual AddrRangeList getAddrRanges() const = 0;
 
     /**
      * We let the master port do the work, so these don't do anything.
+     *
+     * @ingroup api_port
```

Comment:
These should be part of the parent class API, not the API here. Again, let Bobby correct me if I'm wrong here.

Reply:
I'm trying to be consistent here. I assume that bind() and unbind() of Master are also not in the Port API.